packages/quantic/force-app/main/translations/es.translation-meta.xml

```diff
@@ -0,0 +1,891 @@
+    <label>Contraer la faceta {{0}}</label>
+    <name>quantic_CollapseFacet</name>
+  </customLabels>
+  <customLabels>
+    <label>Expandir la faceta {{0}}</label>
+    <name>quantic_ExpandFacet</name>
+  </customLabels>
+  <customLabels>
+    <label>Contraer {{0}}</label>
+    <name>quantic_Collapse</name>
+  </customLabels>
+  <customLabels>
+    <label>Expandir {{0}}</label>
+    <name>quantic_Expand</name>
+  </customLabels>
+  <customLabels>
+    <label>Documentación</label>
+    <name>quantic_Documentation</name>
+  </customLabels>
+  <customLabels>
+    <label>Mensaje</label>
+    <name>quantic_Message</name>
+  </customLabels>
+  <customLabels>
+    <label>Video</label>
+    <name>quantic_Video</name>
+  </customLabels>
+  <customLabels>
+    <label>Recomendado</label>
+    <name>quantic_Recommended</name>
+  </customLabels>
+  <customLabels>
+    <label>Destacado</label>
+    <name>quantic_Featured</name>
+  </customLabels>
+  <customLabels>
+    <label>Mín</label>
+    <name>quantic_Min</name>
+  </customLabels>
+  <customLabels>
+    <label>Máx</label>
+    <name>quantic_Max</name>
+  </customLabels>
+  <customLabels>
+    <label>Introduzca un valor numérico mínimo para la faceta {{0}}</label>
+    <name>quantic_NumberInputMinimum</name>
+  </customLabels>
+  <customLabels>
+    <label>Introduzca un valor numérico máximo para la faceta {{0}}</label>
+    <name>quantic_NumberInputMaximum</name>
+  </customLabels>
+  <customLabels>
+    <label>Filtrar</label>
+    <name>quantic_Apply</name>
+  </customLabels>
+  <customLabels>
+    <label>Aplicar los filtros manuales de periodo para la faceta {{0}}</label>
+    <name>quantic_TimeframeInputApply</name>
+  </customLabels>
+  <customLabels>
+    <label>Aplicar los filtros manuales numéricos para la faceta {{0}}</label>
+    <name>quantic_NumberInputApply</name>
+  </customLabels>
+  <customLabels>
+    <label>Este valor debe ser menor o igual a {{0}}</label>
+    <name>quantic_MessageWhenRangeOverflow</name>
+  </customLabels>
+  <customLabels>
+    <label>Este valor debe ser mayor o igual a {{0}}</label>
+    <name>quantic_MessageWhenRangeUnderflow</name>
+  </customLabels>
+  <customLabels>
+    <label>Cerrar</label>
+    <name>quantic_Close</name>
+  </customLabels>
+  <customLabels>
+    <label>Abrir vista previa</label>
+    <name>quantic_OpenPreview</name>
+  </customLabels>
+  <customLabels>
+    <label>Vista previa no disponible</label>
+    <name>quantic_NoPreviewAvailable</name>
+  </customLabels>
+  <customLabels>
+    <label> :</label>
+    <name>quantic_Colon</name>
+  </customLabels>
+  <customLabels>
+    <label>Última hora</label>
+    <name>quantic_PastHour</name>
+  </customLabels>
+  <customLabels>
+    <label>{{0}} últimas horas</label>
+    <name>quantic_PastHour_plural</name>
+  </customLabels>
+  <customLabels>
+    <label>Último día</label>
+    <name>quantic_PastDay</name>
+  </customLabels>
+  <customLabels>
+    <label>{{0}} últimas días</label>
```

Comment:
Incorrect adjective agreement: últimas días should be últimos días (because día is masculine).

```suggestion
    <label>{{0}} últimos días</label>
```